src/esp/bindings/CMakeLists.txt

```diff
@@ -14,6 +14,10 @@ target_link_libraries(habitat_sim_bindings
     sensor
 )
 
+if(WITH_GL_TENSOR)
+  target_compile_definitions(habitat_sim_bindings PRIVATE __ESP_WITH_GL_TENSOR__)
```

Comment:
I'd stronly advise against using doubly-underscored identifiers. `__Uppercase` as well as `_Uppercase` is reserved by the standard, `_lowercase` is safe.

Could it be just `ESP_WITH_GL_TENSOR`? Or if ESP isn't unique enough then `HABITAT_ESP_WITH_GL_TENSOR`? That would make it easier to grep for as well I think.